src/commands/Minion/sawmill.ts

```diff
@@ -0,0 +1,121 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+
+import { BotCommand } from '../../lib/BotCommand';
+import { Activity, Tasks, Time } from '../../lib/constants';
+import { hasGracefulEquipped } from '../../lib/gear/functions/hasGracefulEquipped';
+import { Planks } from '../../lib/minions/data/planks';
+import { minionNotBusy, requiresMinion } from '../../lib/minions/decorators';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import { SkillsEnum } from '../../lib/skilling/types';
+import { SawmillActivityTaskOptions } from '../../lib/types/minions';
+import { formatDuration, itemNameFromID, stringMatches, toKMB } from '../../lib/util';
+import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			altProtection: true,
+			oneAtTime: true,
+			cooldown: 1,
+			usage: '<quantity:int{1}|name:...string> [plankName:...string]',
+			aliases: ['saw'],
+			usageDelim: ' '
+		});
+	}
+
+	@requiresMinion
+	@minionNotBusy
+	async run(msg: KlasaMessage, [quantity, plankName = '']: [null | number | string, string]) {
+		if (typeof quantity === 'string') {
+			plankName = quantity;
+			quantity = null;
+		}
+
+		const plank = Planks.find(
+			plank =>
+				stringMatches(plank.name, plankName) ||
+				stringMatches(plank.name.split(' ')[0], plankName)
+		);
+
+		if (!plank) {
+			return msg.send(
+				`Thats not a valid log to cut. Valid planks are **${Planks.map(
```

Comment:
Change `Thats not a valid log to cut` to `That's not a valid type of plank to make.`